src/common-utils/main.sh

```diff
@@ -146,6 +146,7 @@ install_debian_packages() {
 # RedHat / RockyLinux / CentOS / Fedora packages
 install_redhat_packages() {
```

Comment:
Can you bump the minor version in here?

https://github.com/devcontainers/features/blob/25942c4359491a0ead6cb755d6ce41e41ee0be05/src/common-utils/devcontainer-feature.json#L3